network.go

```diff
@@ -89,20 +97,29 @@ func (n *Network) initialCleanup(ctx context.Context) {
 		for _, cc := range containers {
 			for _, nnn := range cc.NetworkSettings.Networks {
 				if nnn.NetworkID == nn.ID {
-					if err = n.cli.ContainerRemove(ctx, cc.ID, types.ContainerRemoveOptions{
-						RemoveVolumes: true,
-						Force:         true,
-					}); err != nil {
-						n.t.Fatalf("container removal failure: %s", err.Error())
+					if mapHasLabel(cc.Labels, "owner=testingdock") {
+						if err = n.cli.ContainerRemove(ctx, cc.ID, types.ContainerRemoveOptions{
+							RemoveVolumes: true,
+							Force:         true,
+						}); err != nil {
+							n.t.Fatalf("container removal failure: %s", err.Error())
+						}
+						printf("(setup ) %-25s (%s) - network endpoint removed: %s", nn.Name, nn.ID, cc.Names[0])
+					} else {
+						n.t.Fatalf("container with ID %s already exists, but wasn't started by tesingdock, aborting!", cc.ID)
 					}
-					printf("(setup ) %-25s (%s) - network endpoint removed: %s", nn.Name, nn.ID, cc.Names[0])
 				}
 			}
 		}
-		if err = n.cli.NetworkRemove(ctx, nn.ID); err != nil {
-			n.t.Fatalf("network removal failure: %s", err.Error())
+
+		if mapHasLabel(nn.Labels, "owner=testingdock") {
```

Comment:
I can't see `mapHasLabel` being used anywhere with other argument than `"owner=testingdock"`. So maybe it would be better to turn it into `isOwnedByTestingdock` or smth similar?